core/src/toga/widgets/base.py

```diff
@@ -34,16 +36,31 @@ def __init__(
             will be applied to the widget.
         """
         super().__init__(
-            style=style if style else Pack(),
-            applicator=TogaApplicator(self),
+            style=style if style is not None else Pack(),
```

Comment:
This tripped me up for a minute. With the fleshing out of BaseStyle's dunders, a "blank" style (with no properties set) is currently falsey in main branch Travertino. I think that seems reasonable, but — assuming we don't want to explicitly revert that with a custom `__bool__` method — it's something to be aware of.

Reply:
Needing to compare with None rather than relying on Truthiness is an annoyance, to be sure, but don't think it's worth making a special case out of.